api.md

```diff
@@ -1036,6 +1037,50 @@ CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 
 ---
 
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
+
+This function is an analogue of [pg_create_restore_point()](https://www.postgresql.org/docs/current/functions-admin.html) designed
```

Comment:
The rest of the API doc doesn't use function parentheses after the function name during referring.
You can also use code inside reference. I suggest:
```suggestion
This function is an analogue of [`pg_create_restore_point`](https://www.postgresql.org/docs/current/functions-admin.html) designed
```